.github/workflows/build-deploy-frontend.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Install NPM packages
         run: npm ci
       - name: Build project
-        run: npm run build 
+        run: VITE_API_URL=https://reportvision-ocr-api-dev.azurewebsites.net/ npm run build
```

Comment:
hard coding just for now to get dev working, will leave it to @derekadombek to parameterize